Cargo.toml

```diff
@@ -1,33 +1,38 @@
 [package]
-name = "ingo-blaze"
-version = "0.3.0"
-edition = "2021"
-authors = [ "Ingonyama" ]
+authors = ["Ingonyama"]
 description = "Library for ZK acceleration on Xilinx FPGAs."
+edition = "2021"
 homepage = "https://www.ingonyama.com"
+name = "ingo-blaze"
 repository = "https://github.com/ingonyama-zk/blaze"
+version = "0.3.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
```

Comment:
We should probably open an issue for moving any of these that are dev specific to `[dev-dependencies]`

Reply:
Do you mean dependencies which we use for tests like ark library?

Reply:
Yes, exactly